apis/management.cattle.io/v3/rke_types.go

```diff
@@ -23,6 +23,8 @@ type RancherKubernetesEngineConfig struct {
 	Authorization AuthzConfig `yaml:"authorization" json:"authorization,omitempty"`
 	// Enable/disable strict docker version checking
 	IgnoreDockerVersion bool `yaml:"ignore_docker_version" json:"ignoreDockerVersion" norman:"default=true"`
+	// Is this a DinD install?
+	DinD bool `yaml:"dind" json:"dind"`
```

Comment:
its probably a better to add omitempty to the yaml annotation to avoid confusion when the user run `rke config`